docs/src/content/docs/de/getting-started.mdx

```diff
@@ -89,28 +102,25 @@ Da es sich bei Starlight um eine Beta-Software handelt, wird es regelmäßig Upd
 
 Starlight ist eine Astro-Integration und wird wie jede `@astrojs/*`-Integration aktualisiert:
```

Comment:
This sentence is missing the changes from https://github.com/withastro/starlight/pull/1241/files

Reply:
thank you very much for spotting my mistake! I did miss this completely, my bad!